committers.md

```diff
@@ -86,3 +86,45 @@ non-voting project maintainers.
     {% endif %}
   {% endfor %}
 </tbody></table>
+
+### **Becoming a committer**
+
+As described elsewhere, there are many ways we welcome you to
+contribute the Apache Arrow project such as issue reports,
+documentation, tests, and code. Some particularly active contributors
+may also receive an invitation to become a committer.
+
+The PMC regularly adds new committers from the set of active
+contributors. Becoming a committer is a recognition of sustained
+contribution to the project, and comes with the privilege of
+committing changes directly in all Arrow github repositories. Becoming
+a committer is also a significant responsibility, and committers are
+expected to use their status and access to improve the Arrow project
+for the entire community.
+
+Anyone wishing to become a committer can already do all of the things a
+committer does except for merge a PR. While there is no set list of
+requirements, nor a checklist that entitles one to commit privileges,
+typical behaviors include:
+
+* Consistent, significant, and sustained involvement in the Arrow
```

Comment:
Should "involvement" be expanded on? I think we want to be clear that tossing up a bunch of PRs but not otherwise engaging with others' work isn't sufficient, that the significance of the code contributed is not irrelevant but not the top concern.

Reply:
That's better but I don't know that it's enough. What if in addition to this wording change, we swap the order of the following two bullets. and start the next one like: "Contributions are not limited to making pull requests. They include essential project maintenance work such as reviewing pull requests, fixing bugs ..."

Reply:
Great idea. Thanks @nealrichardson -- I took it one step more in 07d15371c67

```
* Contributions beyond pull requests, such as reviewing other pull requests,
  fixing bugs and documentation, triaging issues, answering community
  questions, improving usability, reducing technical debt, helping
  with CI, verifying releases, debugging in strange environments, etc.

```